### ISSUE TITLE
Need to calculate 2 patterns for all java files in parallel

### ISSUE THREAD

**Issue by lyriccoder** [yegor256/aibolit#26]
Need to calculate 2 patterns for all java files in parallel. At the moment it is not performed in parallel.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@lyriccoder[/z](https://www.0crat.com/u/lyriccoder) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/CT5UB7XAP) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#26` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope
